commit 741f16d8e8d350dede7388066eea7bc44b6c5708
Author: Vincent Batts <vbatts@hashbangbash.com>
Date:   Thu Jul 26 13:41:11 2018 -0400

    thirdpary: explicit libdir path
    
    on some hosts (amd64 centos and fedora) the libdir path defaults to
    lib64, though elsewhere the path is expected to be ./lib/
    
    Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

diff --git a/ci/build_container/build_recipes/nghttp2.sh b/ci/build_container/build_recipes/nghttp2.sh
index f4abbc558..cea6ab963 100755
--- a/ci/build_container/build_recipes/nghttp2.sh
+++ b/ci/build_container/build_recipes/nghttp2.sh
@@ -36,6 +36,7 @@ mkdir build
 cd build
 
 cmake -G "Ninja" -DCMAKE_INSTALL_PREFIX="$THIRDPARTY_BUILD" \
+  -DCMAKE_INSTALL_LIBDIR="$THIRDPARTY_BUILD/lib" \
   -DENABLE_STATIC_LIB=on \
   -DENABLE_LIB_ONLY=on \
   ..
